pkg/controller/servicemesh/memberroll/controller_test.go

```diff
@@ -750,10 +777,10 @@ func markControlPlaneReconciled(controlPlane *maistra.ServiceMeshControlPlane, m
 	return controlPlane
 }
 
-func newAppNamespace() *core.Namespace {
+func newAppNamespace(name string) *core.Namespace {
```

Comment:
This can now be renamed to `newNamespace`.

Reply:
Done